src/platforms/96b_carbon/platform.c

```diff
@@ -36,6 +37,11 @@
 
 jmp_buf fatal_error_jmpbuf;
 
+int platform_hwversion(void)
```

Comment:
Is there a particular reason for removing this from the platform header? This will impair the optimiser's ability to optimise this function out to just a value in all call sites

Reply:
Yes, the mismatching declarations for `platform_hwversion()` lead to a compiler warning.
```
platforms/96b_carbon/platform.h:148:19: error: static declaration of 'platform_hwversion' follows non-static declaration
  148 | static inline int platform_hwversion(void)
      |                   ^~~~~~~~~~~~~~~~~~
In file included from include/general.h:44,
                 from command.c:27:
include/platform_support.h:51:5: note: previous declaration of 'platform_hwversion' with type 'int(void)'
   51 | int platform_hwversion(void);
      |     ^~~~~~~~~~~~~~~~~~
```
Otherwise see the blackpill-f4 review response.